src/java/picard/util/IntervalListTools.java

```diff
@@ -32,9 +32,27 @@
  * @author Tim Fennell
  */
 @CommandLineProgramProperties(
-        usage = " General tool for manipulating interval lists, " +
-                "including sorting, merging, padding, uniqueifying, and other set-theoretic operations. Default operation if given one or more inputs is to " +
-                "merge and sort them.  Other options are controlled by arguments.",
+        usage = "General tool for manipulating interval lists, including sorting, merging, padding, customizing, " +
```

Comment:
Change "customizing" back to "uniquifying". I know it's ugly but it is a fairl widely used technical term.